src/Components/Locations.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+import React, { Component } from "react";
+
+export default class Locations extends Component {
+  constructor() {
+    super();
+    this.state = {
+      locations: [],
+      isShown: false,
+    };
+  }
+  handleClick =()=>{
+      this.setState({
+       isShown: !this.state.isShown,
+      })
+  }
+  getLocations = async () => {
+    const { data } = await axios.get(
+      `https://ghibliapi.herokuapp.com/locations`
+    );
+    this.setState({
+      locations: data.map(
+        (loc) =>
+          (loc = { name: loc.name, climate: loc.climate, terrain: loc.terrain })
+      ),
```

Comment:
Is this .map() necessary? It looks to me like you are copying the entire loc object.

Reply:
except for loc.id! I actually think you should hold onto loc.id because it will be useful in your render() method